### ISSUE TITLE
Sample project needs GitHub Actions support

### ISSUE THREAD

**Issue by jfpanisset** [AcademySoftwareFoundation/aswf-sample-project#21]
ASWF projects have moved to using GitHub Actions, this repo needs to be updated to reflect this, and the information about
using Azure Pipelines perhaps moved to a separate file.

**Comment by akarsh786** [comment]
hey can i work on it

**Comment by jfpanisset** [comment]
Of course, all PRs welcome. Ideally would be great to take a look at what the official ASWF projects are doing and try to capture a representative cross section of their GitHub Actions build pipelines.
